ci/scripts/utils/ci_utils.sh

```diff
@@ -155,3 +155,24 @@ function publish_logs() {
     fi
     echo "${URL}"
 }
+
+function cleanup_experiment() {
+    # cleanup_experiment function
+    # This function takes a directory path of an EXPDIR and
+    # gets HOMEDIR value from config.base to
+    # clean up the archived files on disk.
+
+    local PSLOT_PATH
+    local pslot
+    local HOMEDIR
+    local ARCHIVEDIR
+
+    PSLOT_PATH="$1"
+    pslot=$(basename "${PSLOT_PATH}")
+    HOMEDIR=$(grep 'export HOMEDIR=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]"' | envsubst || true) || true
+    ARCHIVEDIR="${HOMEDIR}/archive"
```

Comment:
I think this should be reading in `$ARCDIR` from `config.base`, similar to how you are reading in `$HOMEDIR`. Note that the default ARCDIR is `${NOSCRUB}/archive`, not `${HOMEDIR}/archive`. `$ARCDIR` already includes the pslot, so it won't need to be added to the next line.

Reply:
Yeah, we should have lead with that.

Reply:
Ok updated using ARCDIR.

Reply:
Test failed because **ARCDIR** has an unresolved sub-string `$NOSCRUB` derived from `${HOMEDIR}`

Reply:
Instead of that, why not just feed the value of `$NOSCRUB` in?
```
ARCHIVEDIR=$(grep 'export ARCDIR=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]' | NOSCRUB="${HOMEDIR}" envsubst || true) || true
```

Or if we want to be more rigorous
```
pslot=$(basename "${PSLOT_PATH}")
HOMEDIR=$(grep 'export HOMEDIR=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]' | envsubst || true) || true
NOSCRUB=$(grep 'export NOSCRUB=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]' | HOMEDIR="${HOMEDIR}" envsubst || true) || true
ARCHIVEDIR=$(grep 'export ARCDIR=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]' | NOSCRUB="${NOSCRUB}" envsubst || true) || true
```

Or we could just cut to the chase and source `config.base` and get everything.

Reply:
IMHO "reconstructing" paths from hidden knowledge of their specificity such as interim archive placements is a non-systematic and haphazard approach resulting in a one-off paradigm.  If we really want to be rigorous I would propose a transformation from logs to a systematic and intent-full pedigree framework that could be leveraged for a requirement such as this.